ReactSkia/core_modules/RSkInputEventManager.cpp

```diff
@@ -121,13 +121,21 @@ void RSkInputEventManager::processKey(RskKeyInput &keyInput) {
       return;//don't propagate key further
     }
   }
+  
 #if defined(TARGET_OS_TV) && TARGET_OS_TV
   sendNotificationWithEventType(
       RNSKeyMap[keyInput.key_],
       currentFocused ? currentFocused->getComponentData().tag : -1,
       keyInput.action_, nullptr);
 #endif //TARGET_OS_TV
   spatialNavigator_->handleKeyEvent(keyInput.key_, keyInput.action_);
+  
+  /*Sending Events to the registered callback*/
+  for (auto pair : eventCallbackMap_){
+    auto fcb = pair.second;
+    fcb(RNSKeyMap[keyInput.key_],currentFocused ? currentFocused->getComponentData().tag : -1,
```

Comment:
should we send this event before focus/blur event or after that ?
if we have to send after , then currentFocused tag need to be fetched again , else we will ending up sending event in wrong sequence